readme.txt

```diff
@@ -126,6 +126,10 @@ If you get stuck, you can ask for help in the Plugin Forum.
 
 == Changelog ==
 
+= 5.2.0 - 2021-04-28 =
```

Comment:
This should be `= 5.2.0 - 2021-xx-xx =`, the `xx-xx` gets replaced during the release process:
```suggestion
= 5.2.0 - 2021-xx-xx =
```

Reply:
done